samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -1008,6 +827,53 @@ For example, for a device provisioned with the device ID ``698d4c11-0ccc-4f04-89
 
          west build --board nrf5340dk_nrf5340_cpuapp -p always -- -DSHIELD=nrf7002ek "-DOVERLAY_CONFIG=overlay_nrf7002ek_wifi_no_lte.conf" -DCONFIG_NRF_CLOUD_PROVISION_CERTIFICATES=y -DCONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME=y  "-DCONFIG_NRF_CLOUD_CLIENT_ID=\`"698d4c11-0ccc-4f04-89cd-6882724e3f6f\`""
 
+.. _nrf_cloud_multi_service_build_hardcoded:
+
+Building with hard-coded CA and device credentials
+==================================================
+
+The :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` Kconfig option allows you to use hard-coded CA and device credentials stored in unprotected program memory for connecting to nRF Cloud.
+
+.. important::
+   The :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` Kconfig option is not secure, and should be used only for demonstration purposes!
+   Because this setting stores your device's private key in unprotected program memory, using it makes your device vulnerable to impersonation.
+
+.. note::
+   This is only one of several mutually exclusive ways to install credentials to your device.
+   See :ref:`nrf_cloud_multi_service_provisioning_onboarding` for a list of other methods.
+
+If you are certain you understand the inherent security risks, you can use this option as follows:
+
+1. Follow the instructions under :ref:`nrf_cloud_multi_service_onboard_hardcoded`.
+
+#. Create a :file:`certs` folder directly in the :file:`nrf_cloud_multi_service` folder, and copy :file:`client-cert.pem`, :file:`private-key.pem`, and :file:`ca-cert.pem` files into it.
+
+   Make sure not to place the new folder in the :file:`nrf_cloud_multi_service/src` folder by accident.
+
+   Now, these certificates are automatically used if the :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` Kconfig option is enabled.
+
+#. Build the sample with the :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` Kconfig option enabled and the device ID you created configured.
+
+   This is required for onboarding to succeed.
+
+   Enable the :kconfig:option:`CONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME` Kconfig option and set the :kconfig:option:`CONFIG_NRF_CLOUD_CLIENT_ID` Kconfig option to the device ID.
+
+   For example, if the device ID is ``698d4c11-0ccc-4f04-89cd-6882724e3f6f``:
+
+   .. tabs::
+
+      .. group-tab:: Bash
+
+         .. code-block:: console
+
+           west build --board *your_board* -p always -- -DCONFIG_NRF_CLOUD_PROVISION_CERTIFICATES=y -DCONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME=y -DCONFIG_NRF_CLOUD_CLIENT_ID="698d4c11-0ccc-4f04-89cd-6882724e3f6f"
+
+      .. group-tab:: PowerShell
+
+         .. code-block:: console
+
+            west build --board *your_board* -p always -- -DCONFIG_NRF_CLOUD_PROVISION_CERTIFICATES=y -DCONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME=y  "-DCONFIG_NRF_CLOUD_CLIENT_ID=\`"698d4c11-0ccc-4f04-89cd-6882724e3f6f\`""
+
 Configuring Wi-Fi access point credentials
```

Comment:
Shouldn't this section go under the Configuration?

Reply:
I think it makes more sense in the building & running section since it specifically includes instructions on how to modify the build environment.

I mean, in some sense, **_all_** of the building & running sections are really just configuration sections that have elevated importance